BSP/Simulator/DataGeneration/simulate.py

```diff
@@ -213,10 +215,46 @@ def main():
                     print("Enter the frequency you would like to change the clock to in Hz.")
                     frequency = int(input())
                     PLL.Change_Frequency(frequency)
+                elif choice == 'EEPROM':
+                    errormsg = 0
+                    print("Enter 'all' for all data or 'read' to enter specific address to read.")
+                    print(">>", end="")
+                    choiceEEPROM1 = input()
+                    print("Enter 'raw' to read the raw hex values or 'msg' for the translated error messages.", end="\n")
+                    print("If invalid response is given, default is raw data.")
+                    choiceEEPROM2 = input()
+                    if choiceEEPROM2 == 'raw':
+                        errormsg = 0
+                    elif choiceEEPROM2 == 'msg':
+                        errormsg = 1
```

Comment:
What is `errormsg` being used for? Is this just to keep track of whether the user chooses 'raw' or 'msg'?

Reply:
Yeah, that's all

Reply:
Can you change `errormsg` to something more descriptive? The name seems to imply that the variable holds an error message

Reply:
made it returnErrorCodes